test/kcli/plans/bonding-nics.yml

```diff
@@ -0,0 +1,46 @@
+
+parameters:
+  output_dir: ../_output/test
+
+vm3:
+  image: fedora37
+  numcpus: 2
+  memory: 2048
+  # output logs from following commands can be found in guest path /var/log/cloud-init-output.log
+  cmds:
+  - yum -y install openvswitch NetworkManager-ovs jq nmstate
+  - systemctl start openvswitch
+  - nmcli connection down "cloud-init eth1"
```

Comment:
Since we're installing nmstate anyway, we could use it to configure the bond. That would simplify this test case quite a bit.

Reply:
Great point! I'm not an expert on nmstate, do you know if it can also delete automatic connections? It would be great if you could provide some snippets to start from.

Reply:
NMState is declarative, so if you tell it to bond two connections it should do whatever is necessary to make that happen. The reason we have trouble with the automatic connections in some envs is that we don't actually run NMState, we just lay down nmconnection files generated by NMState.

The NMState docs have an example of creating a bond (and a lot of other things): https://nmstate.io/examples.html#interfaces-bond